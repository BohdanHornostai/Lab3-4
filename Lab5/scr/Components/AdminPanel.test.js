import React from 'react'
import {render, cleanup} from '@testing-library/react'
import AdminPanel from './AdminPanel'

 afterEach(cleanup)
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<AdminPanel />)

    expect(asFragment(<AdminPanel />)).toMatchSnapshot()
   })