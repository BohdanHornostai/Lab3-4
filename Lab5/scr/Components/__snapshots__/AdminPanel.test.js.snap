// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Admin Panel
    </h2>
    <div
      class="cA"
    >
      <button
        type="submit"
      >
        Upload a photo
      </button>
      <button
        type="submit"
      >
        Delete this product
      </button>
      <button
        type="submit"
      >
        Go to my product
      </button>
    </div>
  </div>
</DocumentFragment>
`;
